daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/InterfacesTest.scala

```diff
@@ -274,18 +274,14 @@ class InterfacesTest
       val command = FetchByInterfaceCommand(idI2, idT2, cid1)
       run(command) shouldBe a[Left[_, _]]
     }
-    // TODO https://github.com/digital-asset/daml/issues/11703
-    //   Enable these tests.
-    /*
-      "be unable to fetch T1 (disguised as T2) via interface I1" in {
-        val command = FetchByInterfaceCommand(idI1, idT2, cid1)
-        run(command) shouldBe a[Left[_, _]]
-      }
-      "be unable to fetch T2 (disguised as T1) via interface I1" in {
-        val command = FetchByInterfaceCommand(idI1, idT1, cid2)
-        run(command) shouldBe a[Left[_, _]]
-      }
-     */
+    "be unable to fetch T1 (disguised as T2) via interface I1" in {
+      val command = FetchByInterfaceCommand(idI1, idT2, cid1)
+      run(command) shouldBe a[Left[_, _]]
```

Comment:
I would like that we check the test is setup correctly.
I personally wrote so many buggy test, and sometime broke others without realized it.
Ideally we would first check the fetch with T1 success, checking the 2 template Ids in the error should be OK as well. 

```suggestion
      run(FetchByInterfaceCommand(idI1, idT1, cid1) shouldBe a[Right[_, _]]
      val command = FetchByInterfaceCommand(idI1, idT2, cid1)
      run(command) shouldBe a[Left[_, _]]
```

Reply:
Fetch with T1 succeeds, it is already tested above.

Reply:
My bad. Thanks.

Reply:
I definitely agree with strengthening the check on the test output, so I did that.